cm/example_empty/worker.py

```diff
@@ -1,17 +1,18 @@
 #!/usr/bin/env python3
 import BaseCM.cm_base as cm_base
-
-from geojson import process
+from BaseCM.cm_output import validate
 
 app = cm_base.get_default_app()
 schema_path = cm_base.get_default_schema_path()
 
 
 @app.task(base=cm_base.CMBase, bind=True, schema_path=schema_path)
-def new_cm(self, file: str, params: dict):
-    """New cm to test the integration work flow"""
-    status = process(file)
-    return status
+def cm_empty(self, selection: dict, rasters: list, params: dict):
+    ret = dict()
+    ret["graphs"] = {}
+    ret["geofiles"] = {}
+    ret["values"] = {}
```

Comment:
as it doesn't add extra information we can have a one liner here:
`ret = {"graphs":{}, "geofiles":{}, "values":{}}`

Reply:
I find it easier for the future "implementers" to understand what to do if it is already decomposed and not merged in one line.

Reply:
Agreed due to the future coders level.